src/styles/_normalizer.scss

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
I guess you should better place these changes to the top as it applies to all elements